curriculum/challenges/english/25-front-end-development/quiz-form-validation-with-javascript/66edd3403d7077eece6dc4b6.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
+What happens if `checkValidity()` is called, but a field fails validation?
 
 #### --distractors--
 
-Placeholder distractor 1
+The form submits without error
 
 ---
 
-Placeholder distractor 2
+The input values are cleared
 
 ---
 
-Placeholder distractor 3
+The form closes immediately
 
 #### --answer--
 
-Placeholder answer
+The invalid field is highlighted and a message appears
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which actions will make `checkValidity()` run?
 
 #### --distractors--
 
-Placeholder distractor 1
+Resetting the form
 
 ---
 
-Placeholder distractor 2
+Changing an input value
 
 ---
 
-Placeholder distractor 3
+Clicking an anchor tag
 
 #### --answer--
 
-Placeholder answer
+Submitting the form manually
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of calling `checkValidity()` before submitting a form?
 
 #### --distractors--
 
-Placeholder distractor 1
+To automatically submit the form
 
 ---
 
-Placeholder distractor 2
+To reload the page
 
 ---
 
-Placeholder distractor 3
+To clear all input fields
 
 #### --answer--
 
-Placeholder answer
+To check if all required fields are filled and valid
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the main purpose of `e.preventDefault()?`
 
 #### --distractors--
 
-Placeholder distractor 1
+To reload the form
 
 ---
 
-Placeholder distractor 2
+To clear form data
 
 ---
 
-Placeholder distractor 3
+To reset the form values
 
 #### --answer--
 
-Placeholder answer
+To stop the default action of an event
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+When used on a form's submit event, what does `e.preventDefault()` do?
 
 #### --distractors--
 
-Placeholder distractor 1
+It submits the form
 
 ---
 
-Placeholder distractor 2
+It clears all form fields
 
 ---
 
-Placeholder distractor 3
+It disables the form temporarily
 
 #### --answer--
 
-Placeholder answer
+It prevents the form from being submitted
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In which scenario is `e.preventDefault()` most useful?
 
 #### --distractors--
 
-Placeholder distractor 1
+When handling a button click that submits a form
 
 ---
 
-Placeholder distractor 2
+When closing a modal dialog
 
 ---
 
-Placeholder distractor 3
+When opening a new page
 
 #### --answer--
 
-Placeholder answer
+When submitting a form via AJAX and preventing a page reload
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens if you forget to call `e.preventDefault()` on a form submit event?
 
 #### --distractors--
 
-Placeholder distractor 1
+The form is submitted via AJAX
 
 ---
 
-Placeholder distractor 2
+The form does not submit
 
 ---
 
-Placeholder distractor 3
+The form data is lost
 
 #### --answer--
 
-Placeholder answer
+The form submits and refreshes the page
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which actions can be prevented using `e.preventDefault()`?
```

Comment:
Hmm, this one's a little off. Technically `e.preventDefault()` doesn't prevent *submitting* a form, it prevents the default behaviour *caused* by submitting a form (re: the page refresh and POST call).